cohere/client.py

```diff
@@ -76,31 +72,24 @@ def check_api_key(self) -> Response:
         try:
             res = json.loads(response.text)
         except Exception:
-            raise CohereError(
-                message=response.text,
-                http_status=response.status_code,
-                headers=response.headers)
+            raise CohereError(message=response.text, http_status=response.status_code, headers=response.headers)
         if 'message' in res.keys():  # has errors
-            raise CohereError(
-                message=res['message'],
-                http_status=response.status_code,
-                headers=response.headers)
+            raise CohereError(message=res['message'], http_status=response.status_code, headers=response.headers)
         return res
 
-    def generate(
-        self,
-        prompt: str,
-        model: str = None,
-        num_generations: int = 1,
-        max_tokens: int = 20,
-        temperature: float = 1.0,
-        k: int = 0,
-        p: float = 0.75,
-        frequency_penalty: float = 0.0,
-        presence_penalty: float = 0.0,
-        stop_sequences: List[str] = None,
-        return_likelihoods: str = 'NONE'
-    ) -> Generations:
+    def generate(self,
+                 prompt: str,
+                 model: str = None,
+                 num_generations: int = 1,
+                 max_tokens: int = 20,
+                 temperature: float = 1.0,
+                 k: int = 0,
+                 p: float = 0.75,
+                 frequency_penalty: float = 0.0,
+                 presence_penalty: float = 0.0,
+                 stop_sequences: List[str] = None,
+                 return_likelihoods: str = 'NONE',
+                 truncate: str = 'NONE') -> Generations:
```

Comment:
Since it's an optional param let's not send a value by default, just like we do with `model`
```suggestion
                 truncate: str = none) -> Generations:
```